lib/index.ts

```diff
@@ -63,96 +59,79 @@ export class MapboxStyleSwitcherControl implements IControl
         this.styleButton = document.createElement("button");
         this.styleButton.type = "button";
         this.mapStyleContainer.classList.add("mapboxgl-style-list");
-        for (const style of this.styles)
-        {
+        for (const style of this.styles) {
             const styleElement = document.createElement("button");
             styleElement.type = "button";
             styleElement.innerText = style.title;
             styleElement.classList.add(style.title.replace(/[^a-z0-9-]/gi, '_'));
             styleElement.dataset.uri = JSON.stringify(style.uri);
-            styleElement.addEventListener("click", event =>
-            {
+            styleElement.addEventListener("click", event => {
                 const srcElement = event.srcElement as HTMLButtonElement;
                 this.closeModal();
-                if (srcElement.classList.contains("active"))
-                {
+                if (srcElement.classList.contains("active")) {
                     return;
                 }
-                if (this.events && this.events.onOpen && this.events.onOpen(event))
-                {
+                if (this.events && this.events.onOpen && this.events.onOpen(event)) {
                     return;
                 }
                 const style = JSON.parse(srcElement.dataset.uri!);
                 this.map!.setStyle(style);
                 const elms = this.mapStyleContainer!.getElementsByClassName("active");
-                while (elms[0])
-                {
+                while (elms[0]) {
                     elms[0].classList.remove("active");
                 }
                 srcElement.classList.add("active");
-                if (this.events && this.events.onChange && this.events.onChange(event, style))
-                {
+                if (this.events && this.events.onChange && this.events.onChange(event, style)) {
                     return;
                 }
             });
-            if (style.title === this.defaultStyle)
-            {
+            if (style.title === this.defaultStyle) {
                 styleElement.classList.add("active");
             }
             this.mapStyleContainer.appendChild(styleElement);
         }
         this.styleButton.classList.add("mapboxgl-ctrl-icon");
         this.styleButton.classList.add("mapboxgl-style-switcher");
-        this.styleButton.addEventListener("click", event =>
```

Comment:
Please dont update the coding style of the project

Reply:
ops, sorry, I hope to have reverted to original code style now